src/confdb/gui/ConfDbGUI.java

```diff
@@ -1243,7 +1243,7 @@ public void openMessageLoggerEditor() {
 
 	/** execute Java Code to manipulate Config */
 	public void openJavaCodeExecution() {
-		JavaCodeExecution execute = new JavaCodeExecution(currentConfig);
+		JavaCodeExecution execute = new JavaCodeExecution(currentConfig, importConfig);
```

Comment:
This means in the GUI you first select & open a config, then select import (of another config as if to import paths or similar), and then select JavaCodeExecution?

Reply:
OK, thanks!